demo/os/armv8a/bin/embed

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
The same comment applies to the demo OS files here and to the "installer" files below as well.